docs/conf.py

```diff
@@ -156,7 +156,7 @@ def setup(sphinx):
 # Add any extra paths that contain custom files (such as robots.txt or
 # .htaccess) here, relative to this directory. These files are copied
 # directly to the root of the documentation.
-html_extra_path = ["_static/css"]
+html_extra_path = ["_static/css", "_static/robots.txt"]
```

Comment:
This supposed to be a directory and not files, but if it works...

Reply:
Well, technically it says "paths" so I guess a file counts too :)